contrib/signet/miner

```diff
@@ -225,7 +225,7 @@ def seconds_to_hms(s):
         out = "-" + out
     return out
 
-def next_block_delta(last_nbits, last_hash, ultimate_target, do_poisson):
+def next_block_delta(last_nbits, last_hash, ultimate_target, do_poisson, max_interval):
```

Comment:
have you thought about defaulting `max_interval` to 3600 so this change doesn't break any existing callers of `next_block_delta`? I see below that the arg parser has a default value, so maybe it's a moot point.

Reply:
You want each call site to use the same `max_interval`, so I thought it was less bug prone for missing the value to have an explicit "missing argument" error, rather than an implicit "cap is set to a different time".

All existing callers in the script should already be updated by this patch; it's not really intended for this to provide a stable api that external callers could rely on.